src/components/dialogs/create-contingency-list-dialog.js

```diff
@@ -258,13 +341,21 @@ export const CreateContingencyListDialog = ({ open, onClose }) => {
                         disabled={
                             contingencyListName === '' ||
                             !contingencyNameValid ||
-                            loadingCheckContingencyName
+                            loadingCheckContingencyName ||
+                            (contingencyListType === ContingencyListType.FORM &&
+                                !isCriteriaBasedEquipmentTypeSelected)
                         }
                     >
                         <FormattedMessage id="validate" />
                     </Button>
                 </DialogActions>
             </Dialog>
+            {renderPopup(
```

Comment:
Could we move the renderPopup component in a separated file please,
rename it as a component : **C**onfirmationDialog
remove useless `intl` prop (we could use `FormattedMessage` component instead)
rename `isConfirmationPopupOpen` to `open`
rename setOpenConfirmationPopup to onClose
rename handlePopupConfirmation to onConfirmation
Add a prop titleMessageDescriptorId
Add a prop contentMessageDescriptorId
Translate {'Confirmation'} replacing by `<FormattedMessage id={titleMessageDescriptorId}/>`
Remove the `div` in the JSX wich is useless here